Sprint-2/stretch/till.js

```diff
@@ -4,28 +4,55 @@
 // When this till object is passed to totalTill
 // Then it should return the total amount in pounds
 
-function totalTill(till) {
-  let total = 0;
+//function totalTill(till) {
+ // let total = 0;
 
-  for (const [coin, quantity] of Object.entries(till)) {
-    total += coin * quantity;
-  }
+  //for (const [coin, quantity] of Object.entries(till)) {
+  //  total += coin * quantity;
+  //}
 
-  return `£${total / 100}`;
-}
+  //return `£${total / 100}`;
+//}
 
-const till = {
-  "1p": 10,
-  "5p": 6,
-  "50p": 4,
-  "20p": 10,
-};
-const totalAmount = totalTill(till);
+//const till = {
+ // "1p": 10,
+ // "5p": 6,
+  //"50p": 4,
+ // "20p": 10,
+//};
+//const totalAmount = totalTill(till);
 
 // a) What is the target output when totalTill is called with the till object
+//"£4.40"
+
 
 // b) Why do we need to use Object.entries inside the for...of loop in this function?
+//Object.entries() is used to convert the till object into an array of key-value pairs. The for...of loop then allows us to iterate over each key-value pair.
+
 
 // c) What does coin * quantity evaluate to inside the for...of loop?
+//Inside the for...of loop, coin is a string representing the denomination (e.g., "1p"), and quantity is a number representing the number of coins (e.g., 10).
```

Comment:
1) You observation of the code is correct. Just that the question was asking what `"1p" * 10` evaluate to.

2) The program cannot evaluate `"1p" * 10` to yield the expected value of `10`, but can we make some small change in the code to get the numerical value of 10 from "1p times 10"?